frame-metadata/src/v15.rs

```diff
@@ -50,6 +50,12 @@ pub struct RuntimeMetadataV15 {
 	pub ty: <PortableForm as Form>::Type,
 	/// Metadata of the Runtime API.
 	pub apis: Vec<RuntimeApiMetadata<PortableForm>>,
+	/// The type of the outer `RuntimeCall` enum.
+	pub call_enum_ty: <PortableForm as Form>::Type,
+	/// The type of the outer `RuntimeEvent` enum.
+	pub event_enum_ty: <PortableForm as Form>::Type,
+	/// The type of the outer `RuntimeError` enum.
+	pub error_enum_ty: <PortableForm as Form>::Type,
```

Comment:
And you  know that the error type isn't encoded like this? I know that you introduced this type now, but it doesn't make that much sense....